test/integration/default/inspec/default_spec.rb

```diff
@@ -66,12 +66,14 @@
   end
 
   %w[foo bar].each do |service|
```

Comment:
FAR: I remember that inspec has way to run test depending on the OS, maybe we could use that instead of our "nice" if os.X?

Reply:
Could be investigated for another review yes.

Reply:
OK